tools/Make/board/odroidc4.mk

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright 2023, UNSW
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+# Set up variables for Odroid C4
+# Should be included _before_ toolchain makefile.
+PLATFORM := meson
+
+I2C_DRIV_DIR := ${PLATFORM}
+NET_DRIV_DIR := ${PLATFORM}
+ETH_DRIV := eth_driver_meson.elf
```

Comment:
Why do we define `ETH_DRIV` but not any of the other ELFs?

Reply:
So that it can be used to create a generic `eth_driver.elf` --- see `docs/makefiles.md`